product_docs/docs/biganimal/release/getting_started/creating_a_cluster/index.mdx

```diff
@@ -61,11 +61,21 @@ To update the parameter values, see [Modifying Your Database Configuration Param
 
 
 ## Availability tab
+### High Availability
  Enable or disable high availability using the **High Availability** slide button. High availability is enabled by default.
  When high availability is enabled, clusters are configured with one primary and two replicas with synchronous streaming replication.
  Clusters are configured across availability zones in regions with availability zones. When high availability is disabled, only one instance is provisioned.
  See [Supported Architectures](../../overview/02_high_availability) for more information.
 
+### Backups
+ in the **Backups** section, you can change the default database backup retention period of 30 days using the **Retention Time** controls. You can configure the retention period as follows:
```

Comment:
Instead of "in", it should be "In". Isn't it ?
One more thing is, we should not merge to main as it is blocked by CNP-1406. Once CNP ticket gets merged, we can include in the document, until that we have to keep it open.